css/index.css

```diff
@@ -1 +1,493 @@
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+    font-size: 62.5%;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.testing * {
+    border: 1px solid palevioletred;
+}
+
+.container {
+  max-width: 880px;
+  /* TB LR */
+  margin: 0 auto;
+}
+
+/* header */
+
+header {
+    display: flex;
+    width: 100%;
+}
+
+header nav {
+  display: flex;
+  width: 100%;
+  vertical-align: top;
+  margin-top: 39px;
+  justify-content: space-between;
+  flex-wrap: nowrap;
+  margin-right: 92px;
+}
+
+header nav a {
+  display: inline-block;
+  color: gray;
+  text-decoration: none;
+  font-size: 1.5rem;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header .logo_container {
+  display: flex;
+  margin-top: 25px;
+  width: auto;
+  margin-right: 2px;
+}
+
+.logo_container img{
+	height: 100%;
+	width: auto;
+}
+
+/* cta */
+
+.cta {
+
+}
+
+.cta .cta-text {
+  display: inline-block;
+  vertical-align: top;
+  font-size: 72px;
+  margin-top: 125px;
+  text-align: center;
+  padding: 0px 75px;
+  margin-left: 50px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 13px;
+  margin-left: 4px;
+  padding: 5px 40px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: inline-block;
+  margin-top: 72px;
+  margin-left: 7px;
+}
+
+.main-content {
+	display: flex;
+	flex-wrap: wrap;
+  border-top: 1px solid black;
+  margin-top: 75px;
+  padding-top: 39px;
+  width: 100%;
+}
+
+.main-content h4 {
+	font-size: 1.6rem;
+}
+
+.main-content .top-content {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.main-content .top-content .text-content {
+  width: 50%;
+}
+
+.main-content .text-content p {
+	margin-top: 18px;
+	font-size: 1.6rem;
+}
+
+.middle-image {
+    width: 100%;
+    height: 182px;
+    margin-top: 33px;
+    background-size: contain;
+}
+
+.main-content .bottom-content {
+  margin-top: 33px;
+  width: 100%;
+  display: flex;
+	justify-content: space-between;
```

Comment:
You have some indentation inconsistencies; [prettier](https://prettier.io) is the solution!

Reply:
I've got it installed on your recommendation it's really good definitely the best beautifier I've used.